general/config.hpp

```diff
@@ -72,7 +72,7 @@ class config
    // **************************************************************************
    static inline void enableGpu(const int dev =0) { Get().MfemDeviceSetup(dev); }
    static inline bool gpuEnabled() { return Get().ngpu > 0; }
-   static inline bool gpuDisabled() { return Get().ngpu == 0; }
+   static inline bool gpuDisabled() { return Get().ngpu < 0; }
    static inline bool gpuHasBeenEnabled() { return Get().ngpu >= 0; }
```

Comment:
@jdahm @camierjs , I don't quite understand the following method. 
``static inline bool gpuHasBeenEnabled() { return Get().ngpu >= 0; }``
Should that check be if ``ngpu >0``  ? Then we can adjust the following method 
``static inline bool gpuDisabled() { return Get().ngpu <= 0; }``

Reply:
By default, `ngpu` is negative, we also need a state where the gpu has been used, but we are back to CPU mode (laghos uses that for problem 0).

Reply:
And I'm not sure this is related to the memory manager, is it?